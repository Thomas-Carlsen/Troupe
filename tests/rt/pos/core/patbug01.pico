let val (a,b) = let val _ = printWithLabels 2121 in (2,21) end 
in printWithLabels 222
end

