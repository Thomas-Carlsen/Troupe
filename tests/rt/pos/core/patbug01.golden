2019-03-07T12:08:10.224Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
2121@{}%{}
222@{}%{}
>>> Main thread finished with value: ()@{}%{}
